src/main/resources/db/changelog/changes/04-create-users-table.yaml

```diff
@@ -37,4 +37,9 @@ databaseChangeLog:
               - column:
                   name: shipping_address
                   type: varchar(255)
-
+              - column:
+                  name: is_deleted
+                  type: bit
+                  constraints:
+                    nullable: false
+                    
```

Comment:
This is normal at the training stage, but you should understand that on real projects you will never change existing files, but always create a new one for new changes